web/src/message_fetch.js

```diff
@@ -394,7 +403,7 @@ export function maybe_load_older_messages(opts) {
 
     do_backfill({
         msg_list,
-        num_before: consts.backward_batch_size,
+        num_before: opts.num_before || consts.backward_batch_size,
```

Comment:
We should put the 1000 in `consts.`; the point of that structure is to have all these sizes collected in one place. 

I think the right preparatory refactor is that we should rename `consts.backward_batch_size` and its forward cousin to `consts.narrowed_view_backward_batch_size`... and then this commit can introduce a `consts.recent_view_fetch_more_batch_size` and decide which to use with a new `opts` parameter, like `for_recent_view: true`.